packages/@icon-magic/generate/src/plugins/svg-generate.ts

```diff
@@ -141,14 +141,14 @@ export const svgGenerate: GeneratePlugin = {
             classNames: classNames
           }
         },
+        {
+          removeViewBox: false
```

Comment:
this is my change, make sure you're rebased.

Reply:
fixed now